### PR TITLE
Update dark theme to match design

### PR DESCRIPTION
**Pull request by Sonicrida**
https://mycryptobuilds.com/ui/UI-35-Dark-Mode-Theme-Colors/

Add the updated Dark Theme colors to the Theme.

image

image

image

### CODE REVIEW

File: src/Theme.ts [hunk 1]
```diff
@@ -1,15 +1,18 @@
 import { darken, invert, lighten, modularScale } from 'polished';
 
 const backgroundLight = 'white';
-const backgroundDark = '#21252b';
-export const borderRadius = '0.125em';
+const backgroundDark = '#214772';
+export const borderRadius = '6px';
```
Comment (nickmccurdy) [inline src/Theme.ts]: Do we want to increase the border radius for both light and dark themes? I believe this is 3 times larger, though I don't mind it.

File: src/atoms/Icon/Icon.tsx [hunk 2]
```diff
@@ -65,6 +65,7 @@ export const icons = {
 const StyledInlineSVG = styled(InlineSVG)`
   svg {
     ${size(scale(0))};
+    color: ${props => props.theme.iconColor};
```
Comment (nickmccurdy) [inline src/atoms/Icon/Icon.tsx]: I thought we were using the font color as the icon color by default

File: src/Theme.ts (hunk 1 above)
Comment (nickmccurdy) [inline src/Theme.ts]:
```suggestion
export const borderRadius = '0.125em';
```

Comment (Sonicrida) [reply src/Theme.ts]: Resolved below

**Comment by nickmccurdy** [comment]
I fixed the merge conflicts, but AWS is failing and I'm not sure why.

**Comment by Sonicrida** [comment]
Is this perhaps related to an issue we've had before?

**Pull request merged**